modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1720,6 +1759,86 @@ GAPI_EXPORTS GMat warpAffine(const GMat& src, const Mat& M, const Size& dsize, i
                              int borderMode = cv::BORDER_CONSTANT, const Scalar& borderValue = Scalar());
 //! @} gapi_transform
 
+/** @brief Parses output of SSD network.
+Extracts detection information (box, confidence, label) from SSD output and
+filters it by given confidence and label.
+
+@note Function textual ID is "org.opencv.core.parseSSDWL"
+
+@param in Input CV_32F tensor with {1,1,N,7} dimensions.
+@param in_sz Size to project detected boxes to (size of the input image).
+@param confidence_threshold If confidence of the
+detection is smaller than confidence threshold, detection is rejected.
+@param filter_label If provided (!= -1), only detections with
+given label will get to the output.
+@return a vector of detected boxes and a vector of appropriate labels.
+*/
+GAPI_EXPORTS core::GDetections parseSSD(const GMat& in,
```

Comment:
I'd prefer to see a plain `GArray<>` here - remember this header is a part of documentation and `GDetections` is one more level of indirection. Looking at the `parseSSD()` prototype alone, would you understand how to handle it's return type?

Reply:
Fixed. `std::tuple<GArray<Rect>, GArray<int>>` instead `GDetections`.

Reply:
Is it aligned with that's there in `.cpp`?